LottoDiary/LottoDiary/Sources/Present/Calendar/ViewController/LottoCalendarViewController.swift

```diff
@@ -0,0 +1,410 @@
+    
+
+}
+
+
+// MARK: - DiffableDataSource, CompositionalLayout 관련 함수
+extension LottoCalendarViewController {
+    func configureCollectionView() {
+        let collectionView = UICollectionView(frame: .zero, collectionViewLayout: generateLayout())
+        
+        contentView.addSubview(collectionView)
+        collectionView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        collectionView.delegate = self
+        collectionView.register(LottoCell.self, forCellWithReuseIdentifier: LottoCell.reuseIdentifier)
+        collectionView.backgroundColor = .designSystem(.gray17181D)
+
+        collectionView.isScrollEnabled = false
+        lottosCollectionView = collectionView
+        
+        viewModel.filteredLottos
+            .map { $0.count }
+            .observe(on: MainScheduler.instance)
+            .subscribe(onNext: { count in
+                self.lottosCollectionView.snp.makeConstraints { make in
+                    make.leading.equalToSuperview()
+                    make.top.equalTo(self.calendarHeight + 20) // 캘린더와의 거리
+                    make.width.equalToSuperview()
+                    // 동적으로 데이터가 들어가야함
+                    make.height.equalTo(count * 90 + 20 + 50) // 셀 크기, 헤더(MM월 dd일), 푸터
+                    // bottom으로 contentView를 마무리지음
+                    make.bottom.equalToSuperview()
+                }
+            })
+            .dispose()
+    }
+    
+    func configureDataSource() {
+        dataSource = DataSource(collectionView: lottosCollectionView) {
+            (collectionView: UICollectionView, indexPath: IndexPath, lottoId: Lotto.ID) ->
+            UICollectionViewCell? in
+            
+            guard let cell = collectionView.dequeueReusableCell(
+                withReuseIdentifier: LottoCell.reuseIdentifier,
+                for: indexPath) as? LottoCell
+            else { fatalError("Could not create new cell") }
+            let lotto = self.lotto(for: lottoId)
+            
+            if lotto != nil {
+                cell.purchaseAmount.text = "\(lotto!.purchaseAmount.formattedWithSeparator) 원"
+                cell.winningAmount.text = "\(lotto!.winningAmount.formattedWithSeparator) 원"
+                cell.type = lotto!.type
+            }
+            return cell
+        }
+        
+        let headerRegistration = UICollectionView.SupplementaryRegistration(elementKind: DateHeaderView.elementKind, handler: headerRegistrationHandler)
+        let footerRegistration = UICollectionView.SupplementaryRegistration(elementKind: AddLottoFooterView.elementKind, handler: footerRegistrationHandler)
+        
+        dataSource.supplementaryViewProvider = { [unowned self]
+            (collectionView, elementKind, indexPath) -> UICollectionReusableView? in
+            
+            if elementKind == DateHeaderView.elementKind {
+                
+                // Dequeue header view
+                return self.lottosCollectionView.dequeueConfiguredReusableSupplementary(
```

Comment:
```suggestion
                return collectionView.dequeueConfiguredReusableSupplementary(
```
collectionView로 해당 lottosCollectionView를 받아오니 대체해도 될 것 같습니다.